src/components/Translator.js

```diff
@@ -2,20 +2,22 @@ import React from 'react'
 
 export default (props) => {
   return(
-    <div className="container">
+    <div className="container" id="translatorBox">
       <h1>English => Emoji</h1>
-      <form onSubmit={props.handleSubmit}>
+      <form onSubmit={props.handleSubmit(props.storyID)}>
```

Comment:
i think you need to wrap this in an anonymous function. This will make sure that you can prevent the default action AND pass the correct parameter through

`<form onSubmit={(e) => {
  e.preventDefault()
  props.handleSubmit(props.storyID) 
}}>`